examples/library_ceph/ceph.py

```diff
@@ -36,21 +36,61 @@
 def deploy():
     db.clear()
     resources = vr.create('nodes', 'templates/nodes.yaml', {'count': 2})
-    first_node, second_node = [x for x in resources if x.name.startswith('node')]
+    first_node, second_node = [x for x in resources if x.name.startswith('solar-dev')]
     first_transp = next(x for x in resources if x.name.startswith('transport'))
+    
+    hosts1 = vr.create('hosts_file1', 'resources/hosts_file', {})[0]
```

Comment:
in nodes.yaml we are already creating hosts_file for each resource. You don't need this. Just get created resources in the same way as first_node and second_node